src/pybind/mgr/orchestrator/_interface.py

```diff
@@ -773,7 +773,7 @@ def set_alertmanager_access_info(self, user: str, password: str) -> OrchResult[s
         """set alertmanager access information"""
         raise NotImplementedError()
 
-    def get_prometheus_cert(self, url: str) -> OrchResult[str]:
+    def get_prometheus_cert(self) -> OrchResult[str]:
         """set prometheus target for multi-cluster"""
```

Comment:
while we're here `set prometheus target for multi-cluster` should be `get prometheus cert for multi-cluster` or something?

Reply:
Thanks @adk3798 , updated the PR